olive/model/handler/base.py

```diff
@@ -36,12 +37,13 @@ def __init__(
         model_file_format: ModelFileFormat,
         model_path: OLIVE_RESOURCE_ANNOTATIONS = None,
         model_attributes: Optional[Dict[str, Any]] = None,
+        io_config: Union[Dict[str, Any], IoConfig, str, Callable] = None,
     ):
         self.framework = framework
         self.model_file_format = model_file_format
         self.composite_parent = None
         self.model_attributes = model_attributes
-        self.io_config = None
+        self.io_config = io_config
```

Comment:
I think the simplest solution would be to just remove `self.io_config` in the base class and remove the IoConfigMixin. 

We can either add this mixin to the subclasses that have self.io_config but don't implement `get_io_config`, or implement the method directly in them.

Reply:
Nevermind, io_config is used in almost all sub classes so this is not efficient either.